### DISCUSSION TITLE
直接运行的端口在哪里？

### DISCUSSION THREAD

**Discussion by OrdChaos** [ArtalkJS/Artalk#86] [Q&A] [answered]
rt，直接运行则8080端口空闲，也无法访问。
运行后提示 http server started on [::]:23366
我很疑惑

**Comment by qwqcode** [comment]
能否提供一下你的 Artalk 配置文件，默认端口不是 8080，是提示的 23366

**Comment by xiamuguizhi** [comment, marked as answer]
23366 正常啊。

**Reply by OrdChaos** [reply]
但访问也不行
地址 http://artalk.ordchaos.com:22336/

部分配置文件：
```
# HTTP Server
host: "0.0.0.0"

# 端口
port: 23366

# 加密密钥
app_key: "**************"

# 调试模式
debug: false

# 时区
timezone: "Asia/Shanghai"

# 数据文件
db:
  type: "mysql" # 支持 sqlite, mysql
  #dsn: "./data/artalk-go.db"
  dsn: '********' # MySQL 连接配置

# 日志
log:
  enabled: true # 总开关
  filename: "./data/artalk-go.log" # 日志文件路径

# 允许跨域访问的域名
allow_origins:
  - '*' # '*' 跨域无限制
# - 'http://localhost:8080'

# SSL
ssl:
  enabled: false
  cert_path: ""
  key_path: ""

# 默认站点名
site_default: "序炁的博客"
```

**Reply by qwqcode** [reply]
如果是 Linux，可以试试执行命令 `curl -I http://127.0.0.1:22336/` 检查是否本地能访问

**Reply by qwqcode** [reply]
我知道了，你端口看错了，是 http://artalk.ordchaos.com:23366/ 不是 22336

**Reply by xiamuguizhi** [reply]
> 我知道了，你端口看错了，是http://artalk.ordchaos.com:23366/不是22336

欢迎来到 Arttalk Go！  哈哈 访问ok

**Reply by qwqcode** [reply]
哈哈

**Reply by OrdChaos** [reply]
emmmm我是傻逼

**Reply by OrdChaos** [reply]
然后就是为何docker不行？

**Reply by qwqcode** [reply]
> 然后就是为何docker不行？

也是端口看错了？

**Comment by xiamuguizhi** [comment]
@OrdChaos 你参考一下这篇文章：https://imhan.cn/posts/20220218

或者：https://github.com/ArtalkJS/Artalk/discussions/46

**Reply by OrdChaos** [reply]
docker试过了，完全照着做，没有错误提示，但是就好像没运行一样，同样不能用

**Reply by monsterxcn** [reply]
你具体是怎么做的呢？

docker-compose 的话能不能看看你的 .yml 呢？

命令行直接跑的话能不能看看你的命令具体是怎么写的呢？

docker logs 有没有显示点啥呢？

感觉主要是得排查下端口映射。

----------------

哦还有就是 host 配置是怎么写的呢？

要是 `localhost` `127.0.0.1` 什么的不行可以试试宿主机 IP，就是 `172.17.0.x` 那个
